web/themes/contrib/civictheme/civictheme_library/components/00-base/flyout/flyout.js

```diff
@@ -137,6 +142,11 @@ CivicThemeFlyout.prototype.expand = function () {
   this.el.setAttribute('data-flyout-expanded', true);
   this.panel.setAttribute('aria-hidden', false);
   document.body.style.overflow = 'hidden';
+
+  // Set focus on first link if available.
+  if (this.panel.querySelector('a.ct-link')) {
```

Comment:
Thanks Alex,

To avoid breaking any current implementations, the focus feature will be disabled by default. You can enable it by adding a `data-flyout-focus` attribute to the `data-flyout` element.

If enabled; opening a panel will look in the panel for an element with `data-flyout-focus-in-target` to focus upon. If it doesn't find it, it will fall back to the first `data-flyout-close-trigger`, and if that's not found, the first `data-flyout-close-all-trigger`.

When closing a panel, it will focus upon the openTrigger for the panel it's closing (same functionality as before).